doctr/utils/geometry.py

```diff
@@ -145,22 +145,62 @@ def rotate_abs_boxes(boxes: np.ndarray, angle: float, img_shape: Tuple[int, int]
     return rotated_boxes
 
 
+def remap_boxes(boxes: np.ndarray, orig_shape: Tuple[int, int], dest_shape: Tuple[int, int]) -> np.ndarray:
+    """ Remaps a batch of RotatedBbox (x, y, w, h, alpha) expressed for an origin_shape to a destination_shape.
+    This does not impact the absolute shape of the boxes, but allow to calculate the new relative RotatedBbox
+    coordinates after a resizing of the image.
+
+    Args:
+        boxes: (N, 5) array of RELATIVE RotatedBbox (x, y, w, h, alpha)
+        orig_shape: shape of the origin image
+        dest_shape: shape of the destination image
+
+    Returns:
+        A batch of rotated boxes (N, 5): (x, y, w, h, alpha) expressed in the destination referencial
+
+    """
+
+    if len(dest_shape) != 2:
+        raise ValueError(f"Mask length should be 2, was found at: {len(dest_shape)}")
+    if len(orig_shape) != 2:
+        raise ValueError(f"Image_shape length should be 2, was found at: {len(orig_shape)}")
+    orig_height, orig_width = orig_shape
+    dest_height, dest_width = dest_shape
+    mboxes = boxes.copy()
+    mboxes[:, 0] = ((boxes[:, 0] * orig_width) + (dest_width - orig_width) / 2) / dest_width
+    mboxes[:, 1] = ((boxes[:, 1] * orig_height) + (dest_height - orig_height) / 2) / dest_height
```

Comment:
I'm not sure I follow: the 2D relative coords of the box center should remain the same, or am I missing something? :thinking:

Reply:
Oh my bad, I think I get it now, but we have to specify it's because of the expansion occurring during the rotation :ok_hand:

Reply:
the box should stay at the same position from the center of the box. the test associated is in `test_remap_boxes` with the following snippet: 
```python
pred = geometry.remap_boxes(np.array([[0.25, 0.5, 0.5, 0.33, 0.]]), (80, 30), (160, 30))
target = np.array([[0.375, 0.5, 0.25, 0.1, 0.]])
assert pred.all() == target.all()
```

Reply:
mmmh but that test doesn't work I think, shouldn't it be:
```python
assert np.all(pred == target)
```
or 
```python
assert (pred == target).all()
```

because otherwise that only checks that no value in the array, once casted to a boolean will be False

Reply:
:O ! you are right !! I will change other assertions like that !!

Reply:
We have this kind of assertion in other tests, you guys should maybe have a deeper look at that

Reply:
Hello, this kind of assertion can be found in other places in the code (`test_rotate` and `test_resolve_enclosing_bbox`). You guys should have a look !